libs/coin-modules/coin-xrp/src/network/index.ts

```diff
@@ -72,8 +72,8 @@ export const getTransactions = async (
 ): Promise<XrplOperation[]> => {
   const result = await rpcCall<AccountTxResponse>("account_tx", {
     account: address,
-    ledger_index: "validated",
```

Comment:
Why do you remove this parameter?

Reply:
Thanks, I forget to add the good field. It was an error on the naming of this query param. ledger_index is an integer (the height) and not a string. To get only the ledger who are validated, we need to use validated param who is a boolean (see the docs [here](https://xrpl.org/docs/references/http-websocket-apis/public-api-methods/account-methods/account_tx). I tried to call the node to get transaction with the ledger_index param but I get no transactions. I wonder who it worked before 🤔

Reply:
I am sorry, I made a mistake. The `validated` parameter is not needed anymore. See the docs :

> The account_tx method retrieves a list of validated transactions that involve a given account.